lambdas/db-provision-user-database/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "rds_security_group" {
+  type = string
+  description = "Security group that allow access to the db cluster"
+}
+
+variable "vpc_id" {
+  type = string
+  description = "The VPC the deployment is in"
+}
+
+variable "prefix" {
+  type = string
+  description = "'prefix' for the deployment ecosystem (Core deployment, data persistence deployment, etc)"
+}
+
+variable "subnet_ids" {
+  type = list(string)
+  description = "Subnets to assign to the database provisioning lambda"
+}
+
+variable "rds_user_password" {
+  description = "Password to set for RDS db user"
+  type = string
+}
+
+variable "rds_access_secret_id" {
+  description = "AWS Secrets Manager secret ID containing jsonified db credentials containing at least host, password, port"
```

Comment:
Would recommend the same edit suggested above

Reply:
Seems like this description never got updated?

Reply:
Indeed not, still working through comments, sorry. 

336b0727a